src/OpenColorIO/CPUInfo.cpp

```diff
@@ -107,7 +107,7 @@ CPUInfo::CPUInfo()
             flags |= X86_CPU_FLAG_SSE42;
 
         /* Check OSXSAVE and AVX bits */
-        if (info.reg.ecx & 0x18000000)
+        if ((info.reg.ecx & 0x18000000) == 0x18000000)
         {
             xcr = xgetbv();
             if(xcr & 0x6) {
```

Comment:
hmm, I think this should also be.

```
if((xcr & 0x6) == 0x6) 
```

https://en.wikipedia.org/wiki/Control_register#XCR0_and_XSS

bit 1 SSE (XSAVE feature set enable for MXCSR and XMM regs)
bit 2 AVX (AVX enable, and XSAVE feature set can be used to manage YMM regs)